docs/src/remap/remap.adoc

```diff
@@ -2678,15 +2674,15 @@ thing.
 
 The last step actually sets the tooltable entries in the 'emcStatus'
 structure. The actual action taken depends on whether the
-RANDOM_TOOLCHANGER ini option was set, but at the end of the process
+RANDOM_TOOLCHANGER INI option was set, but at the end of the process
 'toolTable[0]' reflects the tool currently in the spindle.
 
 When that has happened:
 
-. iocontrol signals task to go ahead
-. task tells the interpreter to execute a 'synch()' operation, to see
-  what has changed
-. the interpreter 'synch()' pulls all information from the world model
+. Iocontrol signals task to go ahead.
```

Comment:
I/O control

Reply:
I think this refers to an iocontrol HAL module. Maybe `iocontrol` and I should just not capialize despite the colon?

Reply:
same for `task`.

Reply:
Sure, I only had the title of https://linuxcnc.org/docs/html/config/iov2.html in mind which says "I/O control".
But yeah, just a suggestion...